client/src/main/java/io/obswebsocket/community/client/model/Transition.java

```diff
@@ -3,12 +3,13 @@
 import lombok.Getter;
 import lombok.Setter;
 import lombok.ToString;
+import lombok.experimental.SuperBuilder;
 
 @Getter
 @Setter
 @ToString
+@SuperBuilder
```

Comment:
The GetCurrentTransitionResponse has a Data that extends this.

Reply:
Ok, but why do we need a Builder for that? Where the builder is used?

Reply:
Ah, I see your point :).
I think I added those because I thought all SpecificData classes had them but I must have mistaken them for the Data hierarchy that was there before. Removing them seems to work just fine.